### PR TITLE
Uploading Assignment6,Assignment4Q3

### PR DESCRIPTION
**Pull request by monicapathania**

### CODE REVIEW

**Review by omerana14 — APPROVED** [review]

**Review by omerana14 — APPROVED** [review]
Please prefer clean code

File: Monica/html/Day7Assingment/Home.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
```
Comment (MKadeer17) [inline Monica/html/Day7Assingment/Home.html:10]: Include all your header and nav tag in body tag

File: Monica/html/Day7Assingment/about.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ABOUT</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<header class="container">
```
Comment (MKadeer17) [inline Monica/html/Day7Assingment/about.html]: Include all header and nav tag in body tag

File: Monica/html/Day7Assingment/contact.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CONTACT</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
```
Comment (MKadeer17) [inline Monica/html/Day7Assingment/contact.html:12]: Same here header and nav tag in body tag

File: Monica/html/Day7Assingment/login.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LOGIN</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<header class="container">
```
Comment (MKadeer17) [inline Monica/html/Day7Assingment/login.html:12]: Header and nav tag  in body tag

**Review by MKadeer17 — COMMENTED** [review]
Add all header and nav tag into the body tag.

**Pull request merged**